packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.component.test.js

```diff
@@ -4,18 +4,20 @@ import { shallow } from 'enzyme';
 import FieldTemplate from './FieldTemplate.component';
 
 describe('FieldTemplate', () => {
+	const defaultProps = {
+		isValid: true,
+		description: 'My awesome description',
+		descriptionId: 'myAwesomeField-description',
+		errorId: 'myAwesomeField-error',
+		errorMessage: 'This is wrong o_o',
+		id: 'myAwesomeField',
+		label: 'My awesome label',
+	};
```

Comment:
`defaultProps` spread to avoid redundancy and show only what changes in each case